inginious/frontend/templates/course_admin/stats.html

```diff
@@ -0,0 +1,147 @@
+$def with (course, stats_graph, stats_tasks, stats_users, daterange, error)
+
+$#
+$# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+$# more information about the licensing of this file.
+$#
+
+$var title: $:course.get_name(user_manager.session_language())
+
+$var Column: $:template_helper.call('course_admin_menu',course=course,current='stats')
+
+$def NavbarF():
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="$get_homepath()/course/$course.get_id()">$course.get_name(user_manager.session_language())</a></li>
+            <li class="breadcrumb-item"><a href="$get_homepath()/admin/$course.get_id()" title=$:_('"Administration"') data-toggle="tooltip" data-placement="bottom">
+                <i class="fa fa-user-secret"></i></a></li>
+            <li class="breadcrumb-item active">
+                <a href="#">
+                    <i class="fa fa-area-chart"></i> $:_("Stats") <span class="sr-only">$:_("(current)")</span>
+                </a>
+            </li>
+        </ol>
+    </nav>
+$var Navbar: $:NavbarF()
+
+<h2>$:_("Stats")</h2>
+<form onsubmit="return submitStats();">
+    <div class="form-row">
+        <div class="form-group col-md-6">
+            <label for="stats_from">$:_("Compute stats since")</label>
+            <input type="datetime-local" class="form-control" id="stats_from" placeholder="From" value="$daterange[0].strftime('%Y-%m-%dT%H:%M:%S')">
+        </div>
+        <div class="form-group col-md-6">
+            <label for="stats_to">$:_("until")</label>
+            <input type="datetime-local" class="form-control" id="stats_to" placeholder="To" value="$daterange[1].strftime('%Y-%m-%dT%H:%M:%S')">
+        </div>
+    </div>
+    <button type="submit" class="btn btn-primary">Update</button>
+</form>
+
+<h2>$:_("Stats between {} and {}").format(daterange[0].strftime("%d/%m/%Y %H:%M:%S"), daterange[1].strftime("%d/%m/%Y %H:%M:%S"))</h2>
+
+<h3>$:_("Submissions")</h3>
+<canvas id="canvas"></canvas>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.min.js"></script>

```

Comment:
We were used to provide a copy of each library, so if the licence allows to redistribute it, maybe it would be more coherent to do so.